src/reducers/books.js

```diff
@@ -0,0 +1,19 @@
+import { removeBook, createBook } from '../actions';
+
+const defaultBooks = [
+  { bookId: Math.random(), title: 'Don quijote', category: 'History' },
+  { bookId: Math.random(), title: '100 years of lonely', category: 'History' },
+];
+
+const booksReducer = (state = defaultBooks, action) => {
+  const { book } = action;
+  switch (action.type) {
+    case 'CREATE_BOOK':
+    default:
+      return createBook(book, state);
+    case 'REMOVE_BOOK':
+      return removeBook(book, state);
```

Comment:
- [ ] Please reformulate this function, as it's not optimal. Your `default` case should be the `state`, and inside of the `case 'CREATE_BOOK'` block, you should call `createBook` function, does it make sense?